entity-management-solr/src/main/java/eu/europeana/entitymanagement/solr/model/SolrOrganization.java

```diff
@@ -0,0 +1,265 @@
+package eu.europeana.entitymanagement.solr.model;
+
+import java.util.List;
+import java.util.Map;
+
+import org.apache.solr.client.solrj.beans.Field;
+
+import eu.europeana.entitymanagement.definitions.model.Organization;
+import eu.europeana.entitymanagement.vocabulary.AgentSolrFields;
+import eu.europeana.entitymanagement.vocabulary.ConceptSolrFields;
+import eu.europeana.entitymanagement.vocabulary.EntitySolrFields;
+import eu.europeana.entitymanagement.vocabulary.OrganizationSolrFields;
+
+public class SolrOrganization extends Organization {
+
+	private String payload;
+	
+	public SolrOrganization() {
+		super();
+	}
+
+	public SolrOrganization(Organization organization) {
+		super();
```

Comment:
use super(organization) and remove the use of redundant setters

Reply:
Done.